electrumsv/wallet_database/functions.py

```diff
@@ -315,6 +316,19 @@ def read_account_balance(db: sqlite3.Connection, account_id: int,
     return WalletBalance(*row)
 
 
+@replace_db_context_with_connection
+def read_transaction_block_hashes(db: sqlite3.Connection) -> Set[bytes]:
+    sql = """SELECT block_hash from Transactions"""
+    cursor = db.execute(sql)
+    rows = cursor.fetchall()
+    block_hashes_set: Set[bytes] = set()
```

Comment:
This should not be returning a set, there will be no duplicates produced by this.